person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] Currently, your `@id` will be set to `nill` because it is not passed in the parameter when creating an instance of `Person`. To solve this, kindly set the `@id` to a random number (`Random.rand(1..1000)`) or you pass it in through the parameter. I will recommend the former. 👍